### ISSUE TITLE
where to get output result?

### ISSUE THREAD

**Issue by ranktopgg** [cmderdev/cmder#2803]
### Question

hello, i need some help,, sorry for my english im  notnative speaker,
after i scan on cmder with alot of result,,i cant find where to get the output for full result,, i can copy for some result only,,dont know where i can get full result
thanks you

### Checklist

- [X] I have read the documentation and made sure what I'm looking for isn't present, or is unclear.
- [X] I have searched for similar issues, and either this question wasn't asked before, or I didn't find any that describe my question.

**Comment by daxgames** [comment]
I don't understand the question.

**Comment by ranktopgg** [comment]
> I don't understand the question.

where do i get the log output ? cant find anywhere

**Comment by daxgames** [comment]
Log output for what?  There is no log output for Cmder.

**Comment by DRSDavidSoft** [comment]
Maybe the log output for cmder_diag?

From a `cmd.exe` window (_not cmder:_)

```
%CMDER_ROOT%\vendor\init.bat /d > %USERPROFILE%\Desktop\Cmder_Diag.log
```

@ranktopgg Why do you need the full log?

**Comment by daxgames** [comment]
That is outputting the debug messages to a log.

There is also `cmder_diag.cmd` that outputs some info about the system to screen that can be redirected to a file and attached to an issue report.

**Comment by DRSDavidSoft** [comment]
Yes, also maybe we can add this to the wiki under https://github.com/cmderdev/cmder/wiki/Cmder-troubleshooting for future reference.
